tests/vxlan-decoder-04/test.yaml

```diff
@@ -0,0 +1,27 @@
+requires:
+  min-version: 8
+
+args:
+  - --set decoder.vxlan.enabled=true
+
+pcap: ../vxlan-decoder-02/vxlan.pcap
+
+checks:
+  - filter:
+      count: 1
+      match:
+        event_type: flow
+        proto: "ICMP"
+        flow.pkts_toserver: 4
+        flow.pkts_toclient: 4
+  - filter:
+      count: 4
+      match:
+        event_type: flow
+        dest_port: 4789
+  - filter:
+      count: 4
+      match:
+        event_type: alert
+        tunnel.dest_port: 4789
+        packet_info.linktype_name: RAW
```

Comment:
By the way, @jlucovsky could we have packet_info.ethernet_type for your other PR ? ;-)

Reply:
Yes -- it's on its way: https://github.com/OISF/suricata/pull/11850